samples/ScheduleDemo/Program.cs

```diff
@@ -0,0 +1,153 @@
+            builder.AddTasks(r =>
+            {
+                // Add a demo orchestration that will be triggered by the schedule
+                r.AddOrchestratorFunc<string, string>("DemoOrchestration", async (context, symbol) =>
+                {
+                    var logger = context.CreateReplaySafeLogger("DemoOrchestration");
+                    logger.LogInformation("Getting stock price for: {symbol}", symbol);
+                    try
+                    {
+                        // Get current stock price
+                        decimal currentPrice = await context.CallActivityAsync<decimal>("GetStockPrice", symbol);
+
+                        logger.LogInformation("Current price for {symbol} is ${price:F2}", symbol, currentPrice);
+
+                        return $"Stock {symbol} price: ${currentPrice:F2} at {DateTime.UtcNow}";
+                    }
+                    catch (Exception ex)
+                    {
+                        logger.LogError(ex, "Error processing stock price for {symbol}", symbol);
+                        throw;
+                    }
+                });
+
+                // Add required activities
+                r.AddActivityFunc<string, decimal>("GetStockPrice", (context, symbol) =>
+                {
+                    // Mock implementation - would normally call stock API
+                    return 100.00m;
+                });
+            });
+
+            // Enable scheduled tasks support
+            builder.UseDurableTaskScheduler(connectionString);
+            builder.EnableScheduledTasksSupport();
+        });
+
+        // Configure the client
+        services.AddDurableTaskClient(builder =>
+        {
+            builder.UseDurableTaskScheduler(connectionString);
+            builder.EnableScheduledTasksSupport();
+        });
+
+        // Configure console logging
+        services.AddLogging(logging =>
+        {
+            logging.AddSimpleConsole(options =>
+            {
+                options.SingleLine = true;
+                options.UseUtcTimestamp = true;
+                options.TimestampFormat = "yyyy-MM-ddTHH:mm:ss.fffZ ";
+            });
+        });
+    })
+    .Build();
+
+await host.StartAsync();
+
+IScheduledTaskClient scheduledTaskClient = host.Services.GetRequiredService<IScheduledTaskClient>();
+
+try
+{
+    // list all schedules
+    var schedules = await scheduledTaskClient.ListSchedulesAsync(false);
+    foreach (var schedule in schedules)
+    {
+        var handle = scheduledTaskClient.GetScheduleHandle(schedule.ScheduleId);
+        await handle.DeleteAsync();
+        Console.WriteLine($"Deleted schedule {schedule.ScheduleId}");
+    }
+
+    // Create schedule options that runs every 30 seconds
+    ScheduleCreationOptions scheduleOptions = new ScheduleCreationOptions
+    {
+        OrchestrationName = "DemoOrchestration",
+        ScheduleId = "demo-schedule101",
+        Interval = TimeSpan.FromSeconds(4),
```

Comment:
The comment above says 30 seoncds but the interval here says 4 seconds.

Reply:
updated